assets/shapes.wgsl

```diff
@@ -44,7 +45,7 @@ fn ndot(a: vec2<f32>, b: vec2<f32>) -> f32 {
     return a.x * b.x - a.y * b.y;
 }
 
-fn dot2(a: vec2<f32>) -> f32 {
+fn dot2_(a: vec2<f32>) -> f32 {
```

Comment:
Is there now s built in dot2? Could we use it instead?

Reply:
Could you paraphrase the question?

Shader functions can't end on number anymore due to https://github.com/bevyengine/naga_oil usage.